### DISCUSSION TITLE
Hiding display sound output from macOS?

### DISCUSSION THREAD

**Discussion by stiligFox** [MonitorControl/MonitorControl#884] [Q&A]
Is it possible to use MonitorControl to hide the audio outputs from macOS so they don't show in the menu of sound outputs anymore? I could have sworn I did it once, but I can't remember how! It would be nice to remove them from the list as I will *never* use them!

I have three Viewsonic monitors, all three are connected to an AMD 6900 XT - two are DisplayPort and one is HDMI.

![Screen Shot 2022-01-09 at 14 02 06](https://user-images.githubusercontent.com/28546763/148698645-b4f1531d-a4b8-4729-959a-daf8533d5ef4.png)

**Comment by waydabber** [comment]
No, this was not a MonitorControl feature ever I think. :)
